src/DataTransformer/ValueObjectTransformer.php

```diff
@@ -59,7 +65,37 @@ public function transform($value)
 
     public function reverseTransform($value)
     {
-        return (new ViewDataInstantiator($this->form, $value))->instantiateObject();
+        try {
+            return (new ViewDataInstantiator($this->form, $value))->instantiateObject();
+        } catch (\Throwable $e) {
+            $exceptionHandlers = [];
```

Comment:
I am not too happy with the fact that these lines are basically copy and pasted from the `FormExceptionHandler` class. I think we should extract a class that does this. Though I struggle to find a good name for it. Suggestions welcome.